uft-one-flight-te-reflection-regression/Action1/Script.mts

```diff
@@ -7,7 +7,14 @@ SystemUtil.Run "C:\Program Files (x86)\Mercury Int. Year2000 Demo Server\term_se
 Window("3270/5250 server").Minimize
 wait 2
 SystemUtil.Run "C:\Users\demo\Documents\Micro Focus\Reflection\Reflection Flight.rd3x"
-wait 20
+
+' the following code replaces a wait 20 line that didn't woork to well :-)
```

Comment:
Typo in comments. Assume this should say "The following code replaces a wait 20 line that didn't work too well"